examples/static-html/stoplight-project/index.html

```diff
@@ -8,8 +8,8 @@
     <meta name="description" content="Example site using Stoplight Elements to display docs" />
     <meta name="author" content="Stoplight" />
 
-    <script src="https://unpkg.com/@stoplight/elements-dev-portal/web-components.min.js"></script>
-    <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements-dev-portal/styles.min.css" />
+    <script src="/modules/@stoplight/elements-dev-portal/web-components.min.js"></script>
+    <link rel="stylesheet" href="/modules/@stoplight/elements-dev-portal/styles.min.css" />
```

Comment:
@stoplightio/pierogi-platoon I think this problem has been here for quite a while 😅.

Basically the e2e static-html tests were operating on the published elements version rather than the built version from the PR in question etc.

Reply:
It's a feature, not a bug. ;)

Back in the day Phil really, really *really* wanted each folder in `examples` to be as standalone as possible, so that you could just copy it to a different directory and treat is as a starter for your project.

Therefore each package uses prod stuff, but then when we run the tests, we "swap" (in various ways) prod stuff for local stuff.

Here is a cypress intercept for unpkg stuff for example:

https://github.com/stoplightio/elements/blob/main/cypress/support/commands.js#L5

Reply:
angular, cra and Gatsby swap stuff in package.json on the other hand: https://github.com/stoplightio/elements/blob/main/use-local-elements.sh

Reply:
Ahh, I see. 🤔 hmm - are we sure the cypress thing works? When I spun e2e up locally for the static html example, it seemed to be pulling from prod unpkg (which was causing the tests to fail).

Reply:
It might not work, of course 😂 

I would honestly be happy if we just sticked to using local stuff in examples.

Having this juggling between prod stuff and local stuff made things more complex for very little in exchange.

So I think I would be okay with changes from this PR.